andela/kenya-rc
# chore:stabilize base application

### What does this PR do?
Stabilizes the base application

### Description of tasks to be completed?
- Install deep-equal package
- Install core-js package

### How should this be manually tested?
After cloning of repo, run `meteor npm install` to install dependencies
- run `reaction` to start the application on port 3000
- Open browser and navigate to localhost:3000 to see the homepage

### Any background context you want to provide?
N/A

### What are the relevant pivotal tracker stories?
#154725343

### Screenshots
<img width="1440" alt="screen shot 2018-01-29 at 5 42 55 pm" src="https://user-images.githubusercontent.com/24798364/35522311-3ec3d342-051c-11e8-8e04-26d5a20e877a.png">
